src/Containers/Microsoft.NET.Build.Containers/ImageBuilder.cs

```diff
@@ -33,7 +33,7 @@ internal sealed class ImageBuilder
     /// <summary>
     /// MediaType of the output manifest.
     /// </summary>
-    public string ManifestMediaType => _manifest.MediaType; // output the same media type as the base image manifest.
+    public string ManifestMediaType => _manifest.MediaType!; // output the same media type as the base image manifest.
```

Comment:
Why forcibly un-nullify the mediaType here? Won't this be null in some cases? I'd think the ImageBuilder should flow the nullability up to callers, and only 'fix' the potentially-null mediaType to a default value at the last moment (during `.Build()` perhaps?)

Reply:
oh, okay. i had a different idea - mediaType can be empty in the response content here

Reply:
I'm going to use both - setting mediaType from response content header if empty in manifest, and default to `SchemaTypes.DockerManifestV2` just in case

Reply:
scratch the above. that wouldn't be right because mediaType will never fallback to that default. I implemented my initial idea in a better readable way.